sdk/js/src/token_bridge/getOriginalAsset.ts

```diff
@@ -234,3 +238,32 @@ export async function getOriginalAssetNear(
 
   return retVal;
 }
+
+export async function getOriginalAssetAptos(
+  client: AptosClient,
+  tokenBridgeAddress: string,
+  assetAddress: string,
+): Promise<WormholeWrappedInfo> {
```

Comment:
`WormholeWrappedInfo` sounds like a misnomer, given that it also classifies native assets